docs/index.rst

```diff
@@ -143,22 +143,28 @@ chgres_cube
 """""""""""
 
 | **CLI**: ``uw chgres_cube -h``
-| **API**: ``import uwtools.api.drivers.chgres_cube``
+| **API**: ``import uwtools.api.chgres_cube``
 
 FV3
 """
 
 | **CLI**: ``uw fv3 -h``
-| **API**: ``import uwtools.api.drivers.fv3``
+| **API**: ``import uwtools.api.fv3``
 
-JEDI
-""""
+UPP
+"""
 
-| **CLI**: ``uw jedi -h``
-| **API**: ``import uwtools.api.drivers.jedi``
+| **CLI**: ``uw upp -h``
+| **API**: ``import uwtools.api.upp``
 
+Driver for JEDI
+^^^^^^^^^^^^^^^
 
+| **CLI**: ``uw jedi -h``
+| **API**: ``import uwtools.api.jedi``
 
+JEDI
+""""
```

Comment:
I feel like this is out of place or most likely was missed to be removed.

Reply:
Yes, thank, will fix.